unit-registry/test/pom.xml

```diff
@@ -105,5 +105,11 @@
             <artifactId>registry.unit.remote</artifactId>
             <version>${project.version}</version>
         </dependency>
+        
+        <dependency>
+            <groupId>org.openbase.bco</groupId>
+            <artifactId>authentication.core</artifactId>
```

Comment:
Again: core really needed?

Reply:
Needed because the MockRegistry now starts the Authenticator.